html2asketch/nodeToSketchLayers.js

```diff
@@ -14,7 +14,7 @@ const DEFAULT_VALUES = {
 
 function shadowStringToObject(shadowStr) {
   let shadowObj = {};
-  const matches = shadowStr.match(/^([a-z0-9#., ()]+) ([0-9.]+)px ([0-9.]+)px ([0-9.]+)px ([0-9.]+)px$/i);
+  const matches = shadowStr.match(/^([a-z0-9#., ()]+) ([-]?[0-9.]+)px ([-]?[0-9.]+)px ([-]?[0-9.]+)px ([-]?[0-9.]+)px/i);
```

Comment:
not sure why you removed `$` from the end? I think we want whole string to match.

Reply:
The computedStyle string of the inset shadow is of this type.
'rgba (0, 0, 0, 0.075) 0px 1px 1px 0px inset'
It does not end with 'px'.

Reply:
Then it makes sense to change that regex to `/^([a-z0-9#., ()]+) ([-]?[0-9.]+)px ([-]?[0-9.]+)px ([-]?[0-9.]+)px ([-]?[0-9.]+)px ?(inset)?$/i`. This way you'll also get info about 'inset' as seventh parameter (for inset shadow you'll have `matches[6] === 'inset'`, for regular shadow you'll have `matches[6] === undefined`).